test/extended/networking/network_segmentation.go

```diff
@@ -54,7 +54,7 @@ var _ = Describe("[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:User
 			userDefinedNetworkIPv6Subnet = "2014:100:200::0/60"
 			nadName                      = "gryffindor"
 
-			udnCrReadyTimeout = 5 * time.Second
+			udnCrReadyTimeout = 60 * time.Second
```

Comment:
Please note that as long the `waitForUserDefinedNetworkReady` looks for one of the two possible conditions, it will now wait 1 minute for each of them slowing down the tests.
Its impossible to wait to "one of" conditions using kubectl from what I know.

Anyhow I would leave it 5 seconds, if it takes 1 minutes for status to be updated something is bad happening.

Reply:
ack will change this back.. no there was a bug before causing the poll intervals to act quirky should be fixed now